spec/filters/mutate_spec.rb

```diff
@@ -537,6 +537,53 @@ def pattern_path(path)
     end
   end
 
+  describe "convert booleans to integer" do
+    config <<-CONFIG
+      filter {
+        mutate {
+          convert => [ "[foo][0]", "integer" ]
```

Comment:
I believe that the hash syntax is preferred:

~~~
filter {
  mutate {
    convert => {"[foo][0]" => "integer"}
    convert => {"[foo][1]" => "integer"}
    convert => {"[foo][2]" => "integer"}
    convert => {"[foo][3]" => "integer"}
    convert => {"[foo][4]" => "integer"}
  }
}
~~~

OR:

~~~
mutate {
  convert => {
    "[foo][0]" => "integer"
    "[foo][1]" => "integer"
    "[foo][2]" => "integer"
    "[foo][3]" => "integer"
    "[foo][4]" => "integer"
  }
}
~~~

Reply:
You are right, the latter should be preferred since the former implies that you can slip in some other mutate action between converts such that the other mutate action would trigger between converts converts.... but it doesn't actually work that way (it loops over all the converts (or any single action) all at once.